web/src/lib/types/types.ts

```diff
@@ -381,6 +381,24 @@ export interface Filing {
   agency?: TaxAgency | null;
 }
 
+export interface License {
```

Comment:
it seems like we should call this a LicenseEvent or a LicenseFiling... "License" is pretty generic and I don't think it represents what this is.